src/styles/blocks/card.scss

```diff
@@ -0,0 +1,103 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 32px 16px 16px;
+  box-sizing: border-box;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__product-image {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto;
+  }
+
+  &__product-title {
+    margin-block: 40px 0;
+    font-family: $main-font-family;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    text-align: left;
+    color: $main-color;
```

Comment:
The variable `$main-color` is used here. Make sure this variable is defined in your SCSS setup.